### PR TITLE
npm-find-dupes: add page

### PR DESCRIPTION
**Pull request by SaiTeja-002**
#14000

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/npm-find-dupes.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`npm find-dupes --loglevel={{silent|error|warn|info|verbose}}`
+
+- Output duplicate information in JSON format:
+
+`npm find-dupes --json`
+
+- Limit duplicate search to specific scopes:
+
+`npm find-dupes --scope={{@scope1, @scope2}}`
```
[the remark this answers is not in the document]
Comment (spageektti) [reply pages/common/npm-find-dupes.md:32]:
```suggestion
`npm find-dupes --scope={{@scope1,@scope2}}`
```

[the remark this answers is not in the document]
Comment (SaiTeja-002) [reply pages/common/npm-find-dupes.md:32]: I used the placeholder syntax `{{}}` to indicate that the scopes are user-provided values. When we run the command, we would omit the braces and provide the scopes as a comma-separated list.

`npm find-dupes --scope=@scope1,@scope2`

[the remark this answers is not in the document]
Comment (spageektti) [reply pages/common/npm-find-dupes.md:32]: > I used the placeholder syntax `{{}}` to indicate that the scopes are user-provided values. When we run the command, we would omit the braces and provide the scopes as a comma-separated list.
> 
> `npm find-dupes --scope=@scope1,@scope2`
> 

I think `@sebaastianspeck` meant that there should be no space between `scope1` and `scope2`.

**Pull request merged**